tests/test_partisn.py

```diff
@@ -397,7 +397,11 @@ def write_partisn_input_1D():
     ngroup = 5
 
     # expected output file
-    file_expected = THIS_DIR + '/files_test_partisn/partisn_1D_expected.inp'
+    import platform
+    if platform.python_version().split('.')[0] == '2':
```

Comment:
all these expected files that have a python 2 and python 3 version may not actually be necessary in the end. I need to check if the changes I made for python3 do the same thing in python 2 before I delete the original files (which are the python 2 version). Thanks to dictionaries no longer being ordered, things were being printed in different orders.